src/Tasks/Common/ConflictResolution/ConflictResolver.cs

```diff
@@ -109,13 +113,31 @@ public ConflictResolver(PackageRank packageRank, PackageOverrideResolver<TConfli
                             }
 
                             //  Call ResolveConflict with the new winner and item that previously had an unresolved
-                            //  conflict, so that the correct message will be logged recording that the winner
-                            //  won and why
-                            ResolveConflict(winner, previouslyUnresolvedItem, logUnresolvedConflicts: true);
+                            //  conflict, so that if the previously unresolved conflict loses, the correct message
+                            //  will be logged recording that the winner won and why.  If the conflict can't be
+                            //  resolved, then keep the previously unresolved conflict in the list of unresolved
+                            //  conflicts.
+                            var newWinner = ResolveConflict(winner, previouslyUnresolvedItem, logUnresolvedConflicts: true);
+                            if (newWinner == winner)
+                            {
+                                foundConflict(winner, previouslyUnresolvedItem);
+                            }
+                            else if (newWinner == null)
+                            {
+                                if (newUnresolvedConflicts.Count == 0)
+                                {
+                                    newUnresolvedConflicts.Add(winner);
+                                }
+                                newUnresolvedConflicts.Add(previouslyUnresolvedItem);
+                            }
 
-                            foundConflict(winner, previouslyUnresolvedItem);
+                            
                         }
                         _unresolvedConflictItems.Remove(itemKey);
+                        if (newUnresolvedConflicts.Count > 0)
+                        {
+                            _unresolvedConflictItems[itemKey] = newUnresolvedConflicts;
```

Comment:
I'm not 100% sure I understand why we want to keep the newUnresolvedConflicts around.